### PR TITLE
Update the footer with current version, fix #129

### LINKED ISSUE #129: Update the footer with link to udata and changelog
A follow-up on https://github.com/opendatateam/udata/pull/593

### PR DESCRIPTION
**Pull request by davidbgk**
capture d ecran 2016-10-20 a 22 49 59

Note that the mention of Etalab (redundant with the logo) and the year had both been removed. Discuss 😄

### CODE REVIEW

File: udata_gouvfr/theme/templates/footer.html [hunk 1]
```diff
@@ -75,11 +75,11 @@
...
             <section class="col-xs-9 col-xs-offset-3 col-sm-offset-0 col-sm-2 col-md-2 col-lg-2">
                 <a href="http://www.etalab.gouv.fr">
-                    <img class="logo" alt="logo"
+                    <img class="logo" alt="Etalab"
                         src="{{ theme_static('img/etalab-logo.png') }}"
                         width="152" height="50" />
-                    <p>2015 ETALAB</p>
-                </a>
+                </a><br />
+                <a href="https://github.com/opendatateam/udata/" title="{{ _('Open-source engine') }}">UData</a> <a href="https://udata.readthedocs.io/en/latest/changelog/" title="{{ _('Deployed version') }}">v{{ udata_version() }}</a>
```
Comment (jphnoel) [inline udata_gouvfr/theme/templates/footer.html]: Looks good to me !
Question: where does the udata_version() come from ? I cannot see it in the udata nor udata-gouvfr repositories.

Comment (davidbgk) [inline udata_gouvfr/theme/templates/footer.html]: @jphnoel it comes from a recent addition in udata: https://github.com/opendatateam/udata/pull/593/files#diff-22a5c10e567d2b7e44d566765a43d278R28

**Comment by noirbizarre** [comment]
It needs review and merge on #128 before (to trigger proper CircleCI tests)

**Comment by davidbgk** [comment]
capture d ecran 2016-10-24 a 21 39 38

The new SVG logo is so crisp, it's crazy we didn't do that before!

**Comment by noirbizarre** [comment]
Ah beautiful !!! 😍

**Pull request merged**